src/fsharp/NicePrint.fs

```diff
@@ -1172,6 +1172,71 @@ module private PrintTastMemberOrVals =
                 let resL = stat --- (nameL ^^ wordL (tagPunctuation ":") ^^ (tauL --- (WordL.keywordWith ^^ WordL.keywordSet)))
                 prettyTyparInst, resL
 
+    let prettyLayoutOfMemberShort denv typarInst (v:Val) = 
```

Comment:
Where is this code duplicated from?

Reply:
The function above:  prettyLayoutOfMember

https://github.com/realvictorprm/visualfsharp/blob/5db861ca912998ba12099dc709c965ea8718b823/src/fsharp/NicePrint.fs#L1112

There are some differences, but refactoring would be relatively doable, I expect.